web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -941,6 +955,10 @@ export default defineComponent({
 </template>
 
 <style lang="scss">
+// Handsontable attaches hidden elements to <body> in order to measure text widths. Therefore this
+// cannot be nested inside .harmonizer-style-container or else the measurements will be off.
+@import '~data-harmonizer/lib/dist/es/index';
```

Comment:
This is probably the only thing here that gives me pause. I'd worry about collisions or weird behavior with Vuetify styles. I took a glance at this particular file and it look like everything is pretty well scoped using IDs and classes though, and local testing looked fine as well.

Reply:
I know. I don't like it either which is why I tried to explain why it is apparently necessary in the comment.